build-patch-filter.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+mkdir target
+git diff HEAD~1 HEAD > target/show.patch
+rm -rf patch-filter
```

Comment:
add empty line above this line to separate patch creation and build

Reply:
done